HHS/OPRE-OPS
# refactor: state for wizard

## What changed

Fixes wizard state

## Issue

#2193 #2308 

## How to test

1. Goto Create Agreement wizard
2. complete step 1 and 2
3. make a budgetline on step 3
4. go back to step 2
5. go back to step 3
6. 🟢 budget-lines should persist (state is coming from wizard)

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
